docker/jagger/webclient/start.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+# Docker variables
+IMAGE_NAME="jagger-web-client"
+VERSION_TAG="1.2.6"
+CONTAINER_NAME="jwc"
+HTTP_HOST_PORT=8080
+
+# Container env variables
+JWC_HTTP_PORT=80
+JWC_JDBC_DRIVER=com.mysql.jdbc.Driver
+JWC_JDBC_URL=jdbc:mysql://172.18.128.228:3306/jaggerdb
+JWC_JDBC_USER=root
+JWC_JDBC_PASS=root
+
+echo "In start script for ${IMAGE_NAME}:${VERSION_TAG} Docker image..."
+
+# Step 1. Check and build the image if needed.
+echo "'Check and build the image if needed' phase..."
```

Comment:
Disagree. When you will not enter if cluase below you will just bring meaningless log 'check and build the image ...'. What image? What was result of the check? Was the image built?